src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -3,3 +3,4 @@
 
 [assembly: AssemblyMetadataAttribute("Serviceable", "True")]
 [assembly: InternalsVisibleTo("dotnet.Tests")]
+[assembly: InternalsVisibleTo("Microsoft.DotNet.ProjectJsonMigration.Tests")]
```

Comment:
alright I'll do this today, seems reasonable to me as well, but there will still be an internals visible to here